src/app/dashboard/clients/page.tsx

```diff
@@ -1,7 +1,29 @@
-export default function Clients() {
+import { ClientTable } from "@/components/clientTable/ClientTable";
+import { columns, Client } from "@/components/clientTable/columns";
+
+async function fetchData(): Promise<Client[]> {
+  try {
+    const response = await fetch(`${process.env.NEXT_PUBLIC_URL}/api/clients`, {
+      method: "GET",
+    });
+    const data = await response.json();
+    return data;
+  } catch (e) {
+    console.error(e);
+    return [];
+  }
+}
+
+export default async function Clients() {
```

Comment:
We'll stick with this convention for all top-level page components:
```suggestion
export default async function ClientsPage() {
```